### PR TITLE
Get and set environment vars

### PR DESCRIPTION
**Pull request by stefanfoulis**

### CODE REVIEW

File: divio_cli/cli.py [hunk 1]
```diff
@@ -235,6 +234,60 @@ def project_update(obj):
...
+@click.pass_obj
+def environment_variables(obj, stage, all_, get, set_, unset):
+    """
+    Get and set environment vars.
+
+
+    """
+    check_project_context(obj.project)
+    if set_:
+        vars = dict(set_)
```
Comment (Chive) [inline divio_cli/cli.py]: shadowing builtin `vars`

File: divio_cli/cli.py [hunk 2]
```diff
@@ -235,6 +234,60 @@ def project_update(obj):
...
+    help='set a specific custom environment variable',
+)
+@click.option(
+    '--unset', 'unset',
+    default=None, type=unicode,
+    multiple=True,
+    help='unset an environment variable',
+)
+@click.pass_obj
+def environment_variables(obj, stage, all_, get, set_, unset):
```
Comment (Chive) [inline divio_cli/cli.py]: not a fan of the underscore names.. how about `all_vars`, `set_var`?

File: divio_cli/cloud.py [hunk 3]
```diff
@@ -6,6 +6,7 @@
 import click
 from six.moves.urllib_parse import urlparse
 
+from divio_cli.utils import json_dumps_unicode
```
Comment (Chive) [inline divio_cli/cloud.py]:
```
from . import utils
```

**Comment by vinitkumar** [comment]
@stefanfoulis Any idea how soon it would be merged?

**Comment by stefanfoulis** [comment]
@vinitkumar We're still internally discussing some slight changes to the API. But we'll merge and release very soon.

**Pull request merged**